idd/Energy+.idd.in

```diff
@@ -18784,12 +18784,15 @@ ZoneProperty:UserViewFactors:BySurfaceName,
        \memo (Number of Surfaces)**2 are expected. Any omitted surface pairs will be assumed to have a view factor of zero.
        \extensible:3 - copy last three fields, remembering to remove ;
        \format ViewFactor
-  A1,  \field Space or SpaceList Name
-       \note View factors may be entered for a single space or for a group of spaces in the same enclosure
-       \note by way of Constructon:AirBoundary or open spaces within a zone.
+  A1,  \field Zone or ZoneList or Space or SpaceList Name
+       \note View factors may be entered for a space, zone, group of spaces, or group of zones in the same enclosure
+       \note by way of Constructon:AirBoundary or open spaces within a zone. This name must align with an enclosure
+       \note encompassing the same zones or spaces.
```

Comment:
I haven't said/asked anything because I was confused on the enclosure. There are spaces, zones and enclosures. This text says "zone, group of spaces [we are good to here], or group of zones in the same enclosure by way of Constructon:AirBoundary". So if an air boundary, which is a plane, defines an enclosure, what defines the other enclosing boundaries?

Reply:
> There are spaces, zones and enclosures. This text says "zone, group of spaces [we are good to here], or group of zones in the same enclosure by way of Constructon:AirBoundary". So if an air boundary, which is a plane, defines an enclosure, what defines the other enclosing boundaries?

The surfaces of a space define the starting enclosure boundaries, and an air boundary connects two enclosures, combining them into one larger enclosure. I realize this needs a chapter of documentation with pictures.